docs/tutorials/agent_framework/Chatbot_with_RAG.md

```diff
@@ -0,0 +1,318 @@
+# Topic
+
+> Agent Framework is an experimental feature released in OpenSearch 2.12 and not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](https://github.com/opensearch-project/ml-commons/issues/1161).
+
+> This tutorial doesn't explain what retrieval-augmented generation(RAG) is.
+
+LLM has some known limitation, one is the frozen world to the time when the LLM trained. 
+They have no knowledge about recent events and your internal data. 
```

Comment:
They have no knowledge about recent events and your internal data.  --> They have no knowledge of recent events or your internal data.